src/structure.js

```diff
@@ -68,6 +77,54 @@ Structure.prototype.cursor = function (path) {
   return Cursor.from(self.current, path, changeListener);
 };
 
+Structure.prototype.reference = function (path) {
+  var self = this, pathId = pathString(path);
+  var listenerNs = self._pathListeners[pathId];
```

Comment:
Hmm.. I guess it's fine when `self._pathListeners[pathId]` isn't hugely packed. 

I just naturally think of big O cases :smile: